### PR TITLE
4 new jackets

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
4 new jackets for people to wear. I recommend wearing them with jeans and tshirt for maximum style.

## Why It's Good For The Game
More drip

## Images of changes
image

## Testing
tested

## Changelog
:cl:
add: 4 new jackets available in clothes vendor
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/misc_suits.dm [hunk 1]
```diff
@@ -821,6 +821,74 @@
...
+	name = "varsity jacket"
+	desc = "Stylish jacket for young and invincible."
+	icon_state = "varsity_classic"
+	item_state = "varsity_classic"
+	allowed = list(/obj/item/flashlight,/obj/item/tank/internals/emergency_oxygen,/obj/item/toy,/obj/item/storage/fancy/cigarettes,/obj/item/lighter)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:834]:
```suggestion
```
Not necessary, it inherits from the bomber jacket.

File: code/modules/clothing/suits/misc_suits.dm [hunk 2]
```diff
@@ -821,6 +821,74 @@
...
+	name = "sport varsity jacket"
+	desc = "Stylish jacket for fast and furious."
+	icon_state = "varsity_sport"
+	item_state = "varsity_sport"
+	allowed = list(/obj/item/flashlight,/obj/item/tank/internals/emergency_oxygen,/obj/item/toy,/obj/item/storage/fancy/cigarettes,/obj/item/lighter)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:851]:
```suggestion
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 3]
```diff
@@ -821,6 +821,74 @@
...
+	name = "blood varsity jacket"
+	desc = "Stylish jacket for dangerous and violent."
+	icon_state = "varsity_blood"
+	item_state = "varsity_blood"
+	allowed = list(/obj/item/flashlight,/obj/item/tank/internals/emergency_oxygen,/obj/item/toy,/obj/item/storage/fancy/cigarettes,/obj/item/lighter)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:868]:
```suggestion
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 4]
```diff
@@ -821,6 +821,74 @@
...
+	name = "driver jacket"
+	desc = "Whoever wears such jacket is literally me."
+	icon_state = "driver_jacket"
+	item_state = "driver_jacket"
+	allowed = list(/obj/item/flashlight,/obj/item/tank/internals/emergency_oxygen,/obj/item/toy,/obj/item/storage/fancy/cigarettes,/obj/item/lighter)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	actions_types = list(/datum/action/item_action/zipper)
+	adjust_flavour = "unzip"
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:885]:
```suggestion
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 5]
```diff
@@ -821,6 +821,74 @@
...
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi',
+		)
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:890]:
```suggestion
		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
		)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 6]
```diff
@@ -821,6 +821,74 @@
...
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:873]:
```suggestion
		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
		)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 7]
```diff
@@ -821,6 +821,74 @@
...
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:856]:
```suggestion
		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
		)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 8]
```diff
@@ -821,6 +821,74 @@
...
```
Comment (Henri215) [inline code/modules/clothing/suits/misc_suits.dm:839]:
```suggestion
		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
		)
```

File: code/game/machinery/vendors/generic_vendors.dm [hunk 9]
```diff
@@ -687,6 +687,10 @@
 					/obj/item/clothing/under/misc/pinkhawaiianshirt = 1,
 					/obj/item/clothing/under/misc/bluehawaiianshirt = 1,
 					/obj/item/clothing/under/misc/orangehawaiianshirt = 1,
+					/obj/item/clothing/suit/jacket/varsity = 1,
+					/obj/item/clothing/suit/jacket/varsity_sport = 1,
+					/obj/item/clothing/suit/jacket/varsity_blood = 1,
+					/obj/item/clothing/suit/jacket/driver = 1,
```
Comment (Henri215) [inline code/game/machinery/vendors/generic_vendors.dm:693]: Do you consider them to be costumes? If not then clothesmate would be the place.

**Review by Henri215 — COMMENTED** [review]
image

These pixels, on the vox version, are intended?

**Comment by ATP-Engineer** [comment]
https://en.wikipedia.org/wiki/Ryan_Gosling

**Comment by DGamerL** [comment]
Are these partially invisible like the syndie bomber jacket?

File: code/game/machinery/vendors/generic_vendors.dm (hunk 9 above)
Comment (McRamon) [reply code/game/machinery/vendors/generic_vendors.dm:693]: yeah i thought its not exactly regular clothes

**Comment by McRamon** [comment]
> image
> 
> These pixels, on the vox version, are intended?

yes


> Are these partially invisible like the syndie bomber jacket?

you can check for yourself in icondiff

**Review by Henri215 — APPROVED** [review]
LGTM

**Comment by LiberatedWaveMan** [comment]
anu-patten-2005-04
DO YOU LIKE HURTING OTHER PEOPLE?

**Review by Henri215 — CHANGES_REQUESTED** [review]
image
At least on that drask clothing some pixels are not symmetrical (worth a look on others too).

image
On humans and vox north direction varsity_classic sprite has this rogue black pixel.

File: code/modules/clothing/suits/misc_suits.dm [hunk 10]
```diff
@@ -821,6 +821,50 @@
...
+/obj/item/clothing/suit/jacket/varsity
+	name = "varsity jacket"
+	desc = "Stylish jacket for young and invincible."
+	icon_state = "varsity_classic"
+	item_state = "varsity_classic"
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
+		)
```
Comment (Contrabang) [inline code/modules/clothing/suits/misc_suits.dm:833]: The last ) should match the tabbed line where the list first started
```suggestion
	)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 11]
```diff
@@ -821,6 +821,50 @@
...
+/obj/item/clothing/suit/jacket/varsity_sport
+	name = "sport varsity jacket"
+	desc = "Stylish jacket for fast and furious."
+	icon_state = "varsity_sport"
+	item_state = "varsity_sport"
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
+		)
```
Comment (Contrabang) [inline code/modules/clothing/suits/misc_suits.dm:844]:
```suggestion
	)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 12]
```diff
@@ -821,6 +821,50 @@
...
+/obj/item/clothing/suit/jacket/varsity_blood
+	name = "blood varsity jacket"
+	desc = "Stylish jacket for dangerous and violent."
+	icon_state = "varsity_blood"
+	item_state = "varsity_blood"
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
+		)
```
Comment (Contrabang) [inline code/modules/clothing/suits/misc_suits.dm:855]:
```suggestion
	)
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 13]
```diff
@@ -821,6 +821,50 @@
...
+/obj/item/clothing/suit/jacket/driver
+	name = "driver jacket"
+	desc = "Whoever wears such jacket is literally me."
+	icon_state = "driver_jacket"
+	item_state = "driver_jacket"
+
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi'
+		)
```
Comment (Contrabang) [inline code/modules/clothing/suits/misc_suits.dm:866]:
```suggestion
	)
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by S34NW — APPROVED** [review]

**Comment by Henri215** [comment]
Only thing left is for the sprite issues to be solved.

**Review by Henri215 — APPROVED** [review]
Lewcs good to me now

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by farie82 — APPROVED** [review]

**Pull request merged**